utils/refineMetadata.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable camelcase*/
+const { omit, assign } = require("lodash/fp");
+const uuid = require("uuid/v1");
+const KJUR = require("jsrsasign");
+
+const filterUnacceptableMeta = omit([
+  "survey_url",
+  "tx_id",
+  "iat",
+  "exp",
+  "eq_id",
+  "form_type",
+  "account_service_url"
+]);
+
+const defaultMetadata = questionnaireId => ({
+  tx_id: uuid(),
+  jti: uuid(),
+  iat: KJUR.jws.IntDate.get("now"),
+  exp: KJUR.jws.IntDate.get("now") + 10000,
+  user_id: "UNKNOWN",
+  case_id: uuid(),
+  ru_ref: "12346789012A",
+  ru_name: "ESSENTIAL ENTERPRISE LTD",
+  trad_as: "ESSENTIAL ENTERPRISE LTD",
+  eq_id: questionnaireId,
+  collection_exercise_sid: uuid(),
+  period_id: "201605",
+  form_type: questionnaireId,
+  survey_url: `${process.env.PUBLISHER_URL}${questionnaireId}`
+});
+
+module.exports.refineMetadata = (metadata, questionnaireId) => {
```

Comment:
Could we call this "sanitise" or something similar - I'm not sure refine is the best verb to use.